pkg/package.mk

```diff
@@ -13,6 +13,17 @@ endif
 
 TAG:=$(ORG)/$(IMAGE):$(HASH)$(DIRTY)
 
+REPO?=https://github.com/linuxkit/linuxkit
+ifneq ($(REPO),)
+REPO_LABEL=--label org.label-schema.vcs-url=$(REPO)
+endif
+REPO_DIRTY:=$(shell git update-index -q --refresh && git diff-index --quiet HEAD -- || echo "-dirty")
```

Comment:
Doesn't the same logic we use for `DIRTY` apply here, ie we don't care if the repo is dirty as long as the directory is clean? It seems odd to me to apply tow different notions of dirty